### DISCUSSION TITLE
New Idea

### DISCUSSION THREAD

**Discussion by ClosetCreeper** [ScratchAddons/ScratchAddons#4550] [Ideas]
Could you make it so that there are text boxes on stage? That you can use.

**Comment by Endermaster3690** [comment]
what????

look how many 😕 you are getting

**Reply by rimopa** [reply]
(😕 is confused)

**Reply by Cool-Scratcher** [reply]
I don't know what this request means. There's already a say block and we can't add new features to non-SA users anyways 😛

**Comment by Chiroyce1** [comment]
New features to Scratch itself will not be added - since people without SA cannot use them, and a vast majority of Scratch users don't use SA.

**Comment by Endermaster3690** [comment]
if you want this in sa, you better explain it
